File: app/src/main/java/com/marknkamau/justjava/ui/cart/CartActivityView.java
package com.marknkamau.justjava.ui.cart;

import com.marknkamau.justjava.models.CartItem;

import java.util.List;

interface CartActivityView {
    void displayCart(List<CartItem> cartItems);
    void displayEmptyCart();
    void displayCartTotal(int totalCost);
}

File: app/src/main/java/com/marknkamau/justjava/ui/cart/CartActivityView.kt
package com.marknkamau.justjava.ui.cart

import com.marknkamau.justjava.models.CartItem

internal interface CartActivityView {
    fun displayCart(cartItems: MutableList<CartItem>)
    fun displayEmptyCart()
    fun displayCartTotal(totalCost: Int)
}
